### PR TITLE
fix: create-api-json (and create-typescript-definitions) now work on any operating system

### PR DESCRIPTION
**Pull request by thomasLechaptois**
Used cross-var (https://github.com/elijahmanor/cross-var) to allow use of environment variables on any operating system.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "standard": "^10.0.0",
     "standard-markdown": "^4.0.0",
     "sumchecker": "^2.0.2",
-    "temp": "^0.8.3"
+    "temp": "^0.8.3",
+	"cross-var": "^1.1.0"
```
Comment (MarshallOfSound) [inline package.json]: Some weird indenting going on here

Comment (thomasLechaptois) [reply package.json]: I don't know what is wrong with the display, but the file seems ok to me:
image

Comment (thomasLechaptois) [reply package.json]: You were right, i replaced tab by spaces and the issue is gone, thanks!

**Comment by zeke** [comment]
From the `cross-var` readme:

> Why don't you use `cross-env`?
>
> `cross-env` is great for scripts that need a particular environment variable set, but isn't intended to fix cross-environment issues when using variables inside an `npm script`

Is ` --version=$npm_package_version` the only env var that's needed? If so, then [cross-env](https://github.com/kentcdodds/cross-env) might be a better fit, given that it's more popular and more actively maintained.

**Comment by codebytere** [comment]
@thomasLechaptois  could you please respond to the above question?

**Pull request closed without merging**